nuxeo-runtime/nuxeo-runtime-stream/src/main/java/org/nuxeo/runtime/avro/AvroSchemaStoreService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * (C) Copyright 2018 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     pierre
+ */
+package org.nuxeo.runtime.avro;
+
+import org.apache.avro.Schema;
+import org.apache.avro.message.SchemaStore;
+
+/**
+ * Holds avro schemas and allows to register them.
+ *
+ * @since 10.2
+ */
+public interface AvroSchemaStoreService extends SchemaStore {
+
+    /**
+     * Register the schema into the SchemaStore.
+     *
+     * @param schema to be registered.
```

Comment:
Note about Javadoc (http://www.oracle.com/technetwork/java/javase/tech/index-137868.html):
- first sentence: Use 3rd person (descriptive) not 2nd person (prescriptive) -> `Registers the schema ...`
- parameters without terminating period.